### PR TITLE
Added typescript definition file for use method

### PR DESCRIPTION
**Pull request by cenkingunlugu**
Solves https://github.com/muratcorlu/lambda-expressless/issues/1 :)

### CODE REVIEW

File: src/lambda-wrapper.d.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+declare module 'lambda-expressless' {
+    declare function use(...handlers: any[]): void;
```
Comment (muratcorlu) [inline src/lambda-wrapper.d.ts]: `use` function expects middleware functions, not `any`. And also return a function in style of lambda handler `(event: LambdaEvent, context: LambdaContext, callback: LambdaHandlerCallback): void`

**Review by muratcorlu — CHANGES_REQUESTED** [review]
I was already planning to add TypeScript support but as a fully covered one. It'll be good to cover all of the objects and types that we use in lambda-expressless. I think we can use some part of those types: https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/express-serve-static-core

File: src/lambda-wrapper.d.ts (hunk 1 above)
Comment (cenkingunlugu) [reply src/lambda-wrapper.d.ts]: Made new commit. Can you please check and resolve if it is ok?

**Review by muratcorlu — APPROVED** [review]

**Pull request merged**